python/CSXCAD/Utilities.pyx

```diff
@@ -25,14 +25,16 @@ def CheckNyDir(ny):
     :param ny: int or str
     :returns: int -- direction as 0/1/2
     """
+    CARTESIAN_COORDINATES = ('x','y','z')
+    CYLINDRICAL_COORDINATES = ('r','a','z')
```

Comment:
Those defines could have been better in CSRectGrid,pyx?
Well not too important...